Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoginBean.java

```diff
@@ -72,7 +72,7 @@
      */
     public Boolean isCorrect = false;
 
-    private static boolean firstTimeLogged = true;
+    private boolean firstTimeLogged = true;
```

Comment:
I haven't looked into the code yet, but I think this was static to lazy initialize something when the first user logs in, but never again after that.  Make sure this change is not going to cause side effects.